src/build_options.rs

```diff
@@ -202,6 +202,12 @@ pub struct BuildOptions {
     #[arg(long)]
     pub zig: bool,
 
+    /// Include debug information in the Wheel.
+    /// Currently only support `.pdb` files with
+    /// the same name as the binary(`exe/dll`) on `msvc` platform
+    #[arg(long)]
```

Comment:
This can't be used together with `--strip`. (Not sure if `clap` supports this kind of setup though)

```suggestion
    #[arg(long, conflicts_with = "strip")]
```

Reply:
On `MSVC`, even if `RUSTFLAGS="-Cstrip=symbols"` is specified, `rustc` will still generate a `.pdb` file, so we can't let it conflict. Not sure how it behaves on `macOS`.

Reply:
But does the `.pdb` file contains useful information?